ingest/importer/spreadsheet/ingest_workbook.py

```diff
@@ -5,6 +5,7 @@
 
 SCHEMAS_WORKSHEET = 'Schemas'
 SPECIAL_TABS = [SCHEMAS_WORKSHEET]
+MAX_ROW_PER_SHEET = 10000
```

Comment:
Could set as env variable from the importer. It would be nice to just redeploy and not do a code hotfix if we have to slightly adjust it.

Reply:
Yea, I wasnt sure about that. there are a lot of constants already hardcoded in the client and wasn't sure if this particular variable needs to come from the broker you mean?

Reply:
I imagine it can be an environment variable in the broker code and then it can be an optional config/parameter to WorkbookImporter class.